packages/api/models/Operator.ts

```diff
@@ -20,19 +13,18 @@ export enum AppEnvAccess {
 export interface Operator {
   display_name: string;
   email: string;
-  access: AccessLevel;
+  access: string;
 }
 
-export const operatorFields: ExhaustiveAttributeMap<Operator> = {
-  display_name: null,
-  email: null,
-  access: null,
-};
-
 export interface PortfolioOperator extends Operator {
-  access: AccessLevel.PORTFOLIO_ADMINISTRATOR;
+  access: PortfolioAccess.PORTFOLIO_ADMINISTRATOR;
```

Comment:
I think you only need to put `PortfolioAccess` and the enums will be implied. Same for the `AppEnvAccess` below.